### PR TITLE
Account integration dependencies and urls, Post create model handling

### PR DESCRIPTION
**Pull request by NoahIles**
added a flow to automatically migrate the models when necessary. Haven't finished with the account integrations wanted to merge everyones work.

### CODE REVIEW

File: app/templates/realdealz/game_list.html [hunk 1]
```diff
@@ -5,29 +5,95 @@
...
+  }
+
+  th.developer, td.developer {
+    width: 800px;
+  }
+
+  th, td {
+  word-wrap: break-word;
+  }
+
```
Comment (nickhib) [inline app/templates/realdealz/game_list.html:32]: could also put these in static and call upon them from there, but this does work

**Review by nickhib — APPROVED** [review]
it looks good i had some minor comments tried to get it to run on my container but im running into issues with that so ill approve

File: app/templates/realdealz/game_list.html (hunk 1 above)
Comment (NoahIles) [reply app/templates/realdealz/game_list.html:32]: I agree I actually didn’t write that part not sure what happened there i might have merged weird or something

File: app/realDealz/models.py [hunk 2]
```diff
@@ -76,32 +58,36 @@ class Game(models.Model):
...
         '''Load the initial library from steamspy'''
         l = lib.Library()
         _games = l.get_top_100()
         for game in _games.keys():
+            _g = _games[game]
+            # If the game is already in the database, skip it
+            if Game.objects.filter(appid=_g['appid']).exists():
+                continue
+            if _g['price'] == 999:  
+                _g['price'] = 0
```
[the remark this answers is not in the document]
Comment (NoahIles) [reply app/realDealz/models.py:78]: Facts

**Comment by NoahIles** [comment]
I do want to make sure you can run it on your end before we merge.

**Pull request merged**